corehq/apps/fixtures/migrations/0008_sqllookuptables.py

```diff
@@ -13,17 +9,4 @@ class Migration(migrations.Migration):
         ('fixtures', '0007_db_cascade'),
     ]
 
-    operations = [
-        migrations.RunPython(
-            LookupTablesCommand.migrate_from_migration,
-            reverse_code=migrations.RunPython.noop,
-        ),
-        migrations.RunPython(
-            LookupTableRowsCommand.migrate_from_migration,
-            reverse_code=migrations.RunPython.noop,
-        ),
-        migrations.RunPython(
-            LookupTableRowOwnersCommand.migrate_from_migration,
-            reverse_code=migrations.RunPython.noop,
-        ),
```

Comment:
Typically when we do this we use `prompt_for_historical_migration` to block migrations of very old databases and display instructions for checking out an old version of the code base to perform the historical migration. However, that process is not straightforward because of https://github.com/dimagi/commcare-hq/pull/32392/commits/d8bf0dbc413ca3d999690921f5c499f60bdda672.

Lookup tables will simply disappear if someone attempts to migrate an database from a version before this migration. Should we remove https://github.com/dimagi/commcare-hq/pull/32392/commits/d8bf0dbc413ca3d999690921f5c499f60bdda672 from this PR and maintain a `prompt_for_historical_migration` here indefinitely?

Reply:
Resolution: replace https://github.com/dimagi/commcare-hq/pull/32392/commits/d8bf0dbc413ca3d999690921f5c499f60bdda672 with https://github.com/dimagi/commcare-hq/pull/32392/commits/c8d587a1364c1e3d28bcaeb1c4d2a38691da7ceb and https://github.com/dimagi/commcare-hq/pull/32392/commits/c55aadd05731b998d2962b0c586f2bf3f4706954